### PR TITLE
refactor: add namespace for ngrx utils stuff

### PR DESCRIPTION
**Pull request by Matejk00**
Part 1 of #5588

### CODE REVIEW

File: projects/schematics/src/migrations/2_0/update-cms-component-state_spec.ts [hunk 1]
```diff
@@ -175,7 +175,7 @@ const ALL_TEST_CASES_CLASS = `
       StateWithCms,
```
Comment (marlass) [inline projects/schematics/src/migrations/2_0/update-cms-component-state_spec.ts:173]: I think there is more things to fix in this particular file. I see EntityLoaderState and LoaderState above.

**Review by marlass — CHANGES_REQUESTED** [review]
Could you add to all removed things from public api migration in removed_api to provide info just to append StateUtils instead? There will be some of those, but should be an easy task ;)

File: projects/schematics/src/migrations/2_0/update-cms-component-state_spec.ts (hunk 1 above)
Comment (Matejk00) [reply projects/schematics/src/migrations/2_0/update-cms-component-state_spec.ts:173]: Done.

File: docs/migration/2_0.md [hunk 2]
```diff
@@ -181,7 +181,14 @@ Support for functions dropped:
 - `CartItemListComponent` no longer requires `FormBuilder`, `FeatureConfigService` and  `CartService`. `CartService` was replaced with corresponding methods from `ActiveCartService`. The deprecated method `isSaveForLaterEnabled()` was removed.
 - `CartItemComponent` no longer uses `FeatureConfigService`. The deprecated method `isSaveForLaterEnabled()` was removed.
 - `PaymentFormComponent` requires new parameter now: `UserAddressService`.
-
+- `LoaderState`, `ProcessesLoaderState`, `EntityState`, `EntityLoaderState`, `EntityProcessesLoaderState` were moved under `StateUtils` namespace.
```
Comment (marlass) [inline docs/migration/2_0.md:187]: Could you move it to separate section
like `### State utility changes`.

It is not that automated as constructor changes, as we only add comments for broken code.

File: projects/schematics/src/migrations/2_0/removed-public-api-data.ts [hunk 3]
```diff
@@ -237,4 +266,178 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
...
+    importPath: SPARTACUS_CORE,
+    comment: `'${OF_LOADER_SUCCESS}' was removed.`,
+  },
+  // projects/core/src/state/utils/loader/loader.helpers.ts
+  {
+    node: OF_LOADER_FAIL,
+    importPath: SPARTACUS_CORE,
+    comment: `'${OF_LOADER_FAIL}' was removed.`,
+  },
+  // projects/core/src/state/utils/entity-loader/entity-loader.selectors.ts
```
Comment (marlass) [inline projects/schematics/src/migrations/2_0/removed-public-api-data.ts]: Those 2 below can be dropped. It won't work as it was previously under a namespace. Info in the migration guide will be enough for those.

**Review by marlass — CHANGES_REQUESTED** [review]

**Review by marlass — APPROVED** [review]

**Pull request merged**